libzeth/mpc/groth16/mpc_hash.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+#ifndef __ZETH_MPC_GROTH16_MPC_HASH_HPP__
+#define __ZETH_MPC_GROTH16_MPC_HASH_HPP__
+
+#include "libzeth/core/hash_stream.hpp"
+
+#include <sodium/crypto_generichash_blake2b.h>
+#include <string>
+
+namespace libzeth
+{
+
+// Hashing for MPC. Streaming and whole-buffer interfaces.
+const size_t MPC_HASH_SIZE = 64;
```

Comment:
This is a size in *bytes*, which contrasts with the other constants defined as "bit strings": https://github.com/clearmatics/zeth/blob/clean-libzeth-conventions/libzeth/zeth_constants.hpp#L18-L24

Reply:
Please rename accordingly to `MPC_HASH_SIZE_BYTES` to remove inconsistencies

Reply:
Renamed constant.